plugins/aws_sts/README.md

```diff
@@ -0,0 +1,20 @@
+# AWS STS Plugin
+
+Inject temporary AWS Role credentials into a deploy environment using [AWS STS](https://docs.aws.amazon.com/STS/latest/APIReference/Welcome.html) and [Assume role](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html).
+
+## Overview
+
+Allows Samson to inject temporary security credentials for an AWS Role. The plugin uses
+the assume role feature of STS and exposes the generated credentials as environment variables:
+  - STS_AWS_ACCESS_KEY_ID
+  - STS_AWS_SECRET_ACCESS_KEY
+  - STS_AWS_SESSION_TOKEN
+
+The plugin authenticates with AWS using the following environment variables:
```

Comment:
maybe say `samson environment variables` to make it clear that they are coming from samsons env